webhooks/tests/test_tasks.py

```diff
@@ -0,0 +1,95 @@
+"""
+Celery tests.
+"""
+# pylint: disable=no-self-use
+import hashlib
+import hmac
+import json
+import mock
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.utils.crypto import constant_time_compare
+from django.utils.encoding import force_bytes
+
+from courses.factories import CourseFactory
+from webhooks.models import Webhook
+from webhooks.tasks import publish_webhook
+
+
+class PublishWebhookTests(TestCase):
```

Comment:
Can you add a test for a failure case, for example where there are two webhooks and the first call fails?